### PR TITLE
cicd: Added ppc64le support to the tests in nightly ci

### PR DESCRIPTION
**Pull request by raja-0940**
Moved tests from config ci module to nightly ci with ppc64le support

### CODE REVIEW

**Review by hdonnay — CHANGES_REQUESTED** [review]
I don't understand what this is doing that it doesn't already.

What's the point, here?

**Comment by raja-0940** [comment]
As part of [PROJQUAY-5377](https://issues.redhat.com/browse/PROJQUAY-5377), Worked on adding ppc64le support to the tests for the config ci module of the Clair.

**Comment by hdonnay** [comment]
Sure, but it already does that. What does this addition do?

**Comment by yussufsh** [comment]
If I am not wrong @raja-0940  is adding "Config CI" tests as part of "Nightly CI" for arch-specific runs. Currently, it's just integration tests running for ppc64le.

**Comment by hdonnay** [comment]
That's not the case. The tests already run for the `github.com/quay/clair/config` module. So what does this addition do that the existing step doesn't?

File: .github/workflows/config-ci.yml [hunk 1]
```diff
@@ -64,17 +64,39 @@ jobs:
...
+          docker run \
+          --rm \
+          --platform ${{ matrix.architecture }} \
+          --mount "type=bind,src=$(go env GOMODCACHE),dst=/go/pkg/mod" \
+          --mount "type=bind,src=$(go env GOCACHE),dst=/root/.cache/go-build" \
+          --mount "type=bind,src=$(pwd),dst=/config" \
+          -w /config \
+          quay.io/projectquay/golang:${{ matrix.go }} \
+          go test \
+          -tags tests \
```
Comment (yussufsh) [inline .github/workflows/config-ci.yml]: Is this tags option needed?

**Comment by hdonnay** [comment]
Will close shortly if there's no explanation on how this doesn't duplicate existing functionality.

**Comment by mpkpersistent** [comment]
Hi @hdonnay,
The code changes are targeted to add ppc64le arch support to the config-ci module.
Pl refer https://issues.redhat.com/browse/PROJQUAY-5377 for more details.

**Comment by hdonnay** [comment]
For the fourth time: the `github.com/quay/clair/config` module has the tests run on all architectures in the existing Nightly CI. I'll close this tomorrow if there's no explanation as to why this is needed.

**Comment by raja-0940** [comment]
Had a group discussion with Hank to explain that config module is not having the ppc64le arch support, hence we have created the PR. 

But PR got rejected citing reason that "github.com/quay/clair/config module has the tests run on all architectures in the existing Nightly CI."

**Pull request closed without merging**